geometry_msgs/Pose pose
string block_colour
string detected_colour
